ui/src/app/userinfo/components/cli-help.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {useState} from 'react';
+
+export const CliHelp = () => {
+    const argoSecure = document.location.protocol === 'https';
+    const argoBaseHref = document
+        .getElementsByTagName('base')[0]
+        .href.toString()
+        .replace(document.location.protocol + '//' + document.location.host + '/', '');
+    const argoToken = decodeURIComponent(document.cookie)
+        .split(';')
+        .map(x => x.trim())
+        .find(x => x.startsWith('authorization='))
+        .replace('authorization=', '');
+
+    const text = `export ARGO_SERVER='${document.location.host}' 
+export ARGO_HTTP1=true  
+export ARGO_SECURE=${argoSecure ? 'true' : 'false'}
+export ARGO_BASE_HREF=${argoBaseHref}
+export ARGO_TOKEN='${argoToken}' 
+export ARGO_NAMESPACE=argo ;# or whatever your namespace is 
+export KUBECONFIG=/dev/null ;# recommended
+
+# check it works: 
+argo list`;
+
+    const [copied, setCopied] = useState(false);
+    return (
+        <div className='white-box'>
+            <h4>Using Your Login With The CLI</h4>
+            <p>Download the latest CLI before you start.</p>
+            <div style={{fontFamily: 'monospace', whiteSpace: 'pre', margin: 20}}>{text.replace(argoToken, '[REDACTED]')}</div>
```

Comment:
we redact the ARGO_TOKEN here

Reply:
Why not simply redact it on the original string? It seems `argoToken` is not used anywhere except to get redacted

Reply:
it made it simpler to have a single un-redacted const than two const